src/webpage.cpp

```diff
@@ -1057,18 +1058,23 @@ QString WebPage::renderBase64(const QByteArray& format)
     return "";
 }
 
-QImage WebPage::renderImage()
+QImage WebPage::renderImage(bool onlyViewport)
```

Comment:
@adeodhar-quark thanks for the feedback. I agree with @ariya too. I'll refactor the PR ASAP, probably adding a separate method with a meaningful name.